lib/database-cleaner.js

```diff
@@ -88,7 +88,13 @@ var DatabaseCleaner = module.exports = function(type, config) {
   };
 
   cleaner['postgresql'] = function(db, callback) {
-    db.query("SELECT table_name FROM information_schema.tables WHERE table_schema = 'public';", function(err, tables) {
+    var schema = config.postgresql.schema || 'public';
+    var schemaPrefix = '';
+    if (schema != 'public') {
```

Comment:
public.[tableName] should work, so maybe you don't need this condition.

also, you can consider "set search_path to [schemaName] then you don't need prefix